src/features/movies/MovieList/movieListSaga.js

```diff
@@ -7,13 +7,16 @@ import {
     selectPageState,
     setMovieList,
     setGenres,
+    setLoading,
 } from "./movieListSlice";
 import { getPopularMovies } from "../../../utils/API/getPopularMovies";
 import { getGenreList } from "../../../utils/API/getGenreList";
 import { processMovieListData } from "../../../utils/API/processMovieListData";
 
 function* fetchMovieListHandler() {
     try {
+        yield put(setLoading(true));
+        yield delay(1000);
```

Comment:
This i faaar to long 😄 if we must 200ms tops, please 😉 But really we do not need that delay at all. Page change will animate regardles, sometimes showing the spinner and sometimes not. Why forcefully delay the transition?

Reply:
Thank you for your comment above; you've convinced me. I've changed my mind and acknowledge that you're right; we shouldn't artificially delay the page transition. I will make the necessary adjustments and implement the updated approach. Thanks! 😊